previews/components/enrolment_confirmation_component_preview.rb

```diff
@@ -0,0 +1,25 @@
+class EnrolmentConfirmationComponentPreview < ViewComponent::Preview
+  def default
+    render(EnrolmentConfirmationComponent.new(
+      current_user: User.last.presence,
```

Comment:
Do we need presence on these user calls?

We also have no preview for the user not being logged in?